Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml

```
@@ -0,0 +1,64 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: OpenPhish_v2
+  version: -1
+configuration:
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Database refresh interval (hours)
+  name: fetchIntervalHours
+  defaultvalue: 1
+  type: 0
+  required: false
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+description: OpenPhish uses proprietary Artificial Intelligence algorithms to automatically
+  identify zero-day phishing sites and provide comprehensive, actionable, real-time
+  threat intelligence.
+name: OpenPhish_v2
```

New feature(integrations)s are only from 5.0
```suggestion
fromversion: 5.0.0
name: OpenPhish_v2
```

---

my bad